src/server/services/controllers/accountController.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var models = require('../../db/schema');
+
+function sendError(errorCode, res, err, message) {
+    res.status(errorCode).json({
+        title: message,
+        message: err.message,
+        error: err
+    });
+}
+exports.getAllAccounts = function(req, res, next) {
+    console.log('getAllAccounts');
+    models.getAllAccounts(function(err, accounts) {
+        if (err) {
+            sendError(400, res, err, 'Failed on getAllAccounts');
+        }
+        res.json(accounts);
+    });
+}
+
+exports.getAccountById = function(req, res, next) {
+    console.log('getAccountById ' + req.params.id);
```

Comment:
Similarly a little null-check here would be good or else a LoDash deep get on `req.params.id`.